spring-rabbit/src/main/java/org/springframework/amqp/rabbit/support/micrometer/MessageSenderContext.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.amqp.rabbit.support.micrometer;
+
+import org.springframework.amqp.core.Message;
+
+import io.micrometer.observation.transport.SenderContext;
+
+/**
+ * {@link SenderContext} for {@link Message}s.
+ *
+ * @author Gary Russell
+ * @since 3.0
+ *
+ */
+public class MessageSenderContext extends SenderContext<Message> {
```

Comment:
I call them the same way in Spring Integration as well.
I wonder if we should think about different names.
For example `RabbitSenderContext` 😄 or `RabbitMessageSenderContext`.
Just to avoid confusion on the end-user side.

Or... since it is apparently your Pr is going to be merged first, I probably need to rename them in Spring Integration... 😉